src/glue.rs.in

```diff
@@ -62,13 +87,36 @@ mod _PartitionConsumer {
             inner: Box::pin(run_block_on(consumer.stream(offset.clone()))?)
         })
     }
+    pub fn stream_with_config(
+        consumer: &PartitionConsumer,
+        offset: &Offset,
+        wasm_module_path: &str
+    ) -> Result<PartitionConsumerStream, FluvioError> {
+        let config_wrapper = ConsumerConfigWrapper::new_config_with_wasm_filter(wasm_module_path);
+        let mut builder = ConsumerConfig::builder();
+        builder.smartmodule(Some(SmartModuleInvocation {
+            wasm: SmartModuleInvocationWasm::AdHoc(config_wrapper.wasm_module),
+            kind: SmartModuleKind::Filter,
+            params: Default::default()
+        }));
+        let config = builder.build().expect("Failed to create config");
```

Comment:
`expect` should be removed from the non-test code.   you can convert to FluvioError by doing something like this:
```
let config = builder.build().map_err(|err| FluvioError::Other(err.to_string())?;
```

Reply:
All actioned and removed